auto_cpufreq/core.py

```diff
@@ -267,13 +267,23 @@ def turbo(value: bool = None):
     """
     p_state = Path("/sys/devices/system/cpu/intel_pstate/no_turbo")
     cpufreq = Path("/sys/devices/system/cpu/cpufreq/boost")
+    amd_pstate = Path("/sys/devices/system/cpu/amd_pstate/status")
+    scaling_driver = Path("/sys/devices/system/cpu/cpu0/cpufreq/scaling_driver")
 
     if p_state.exists():
         inverse = True
         f = p_state
     elif cpufreq.exists():
         f = cpufreq
         inverse = False
+    elif amd_pstate.exists() and scaling_driver.exists():
+        amd_value = amd_pstate.read_text().strip()
+        scaling_driver_value = scaling_driver.read_text().strip()
+        if amd_value == "active" and scaling_driver_value == "amd-pstate-epp":
+            print("CPU turbo is controlled by amd-pstate-epp driver")
+        else:
+            print("Warning: CPU turbo is not available")
```

Comment:
These can be removed since this else statement will never be reached. the `cpufreq` Path does exist on `amd-pstate` so it would be caught by the preceding elif statement